app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Issue: Using names as keys in the `people` dictionary can cause issues if names are not unique. 

Suggestion: Consider using a unique identifier for each person or implement a strategy to handle potential name collisions.